How-to guide migration (#165)
vegaprotocol/documentation: docs/howto/amend-order.md

* **PEGGED ORDER**  
A pegged order can be amended in a similar way to a regular limit order. Their `reference`, `offset` and `TIF` values can all be amended. If amending an order cannot be performed in-place, the order will lose time priority in the order book (but will keep its priority in the list of pegged orders). Amends must be done to the pegged order itself, not any limit orders derived from pegged orders. It is **not possible** to amend a regular limit order to be a pegged order, and similarly it is also **not possible** to amend a pegged order to be a regular limit/market order. Please use the `peggedReference` field to amend the pegged order reference and the `peggedOffset` field to amend the pegged order offset distance, both can be amended independently. Sample scripts for submitting and amending a pegged order are available on [GitHub](https://github.com/vegaprotocol/sample-api-scripts/tree/master/submit-amend-pegged-order). 

## How do I amend an order?

If an order is active and was placed with a party for which you have a public/private key pair, you can amend it. Connect to a Vega API server, and *amend an order*:

> ### 1. Log in to wallet and get public key

Reviewer: ```suggestion
### 1. Connect to wallet and get public key
```

> See the section on the [Wallet service](wallet.md) to learn how to log in, list keys and select a public key.  

Reviewer: ```suggestion
See the section on the [Wallet service](wallet.md) to learn how to connect to an existing wallet, list keys and select a public key.  
```
